src/components/modal/v2/parts/views/ProductList/Content.jsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useServerData } from '../../../lib';
+import Icon from '../../Icon';
+import Tile from './Tile';
+import styles from './styles/index.scss';
+
+export const ProductList = ({ content: { instructions, disclosure, tiles }, setProduct, contentBodyRef }) => {
+    const views = useServerData()?.views;
+    const availableTiles = views
+        .map(view => tiles.find(tileContent => tileContent.viewName === view.meta.product))
+        .filter(tile => !!tile);
```

Comment:
```suggestion
    const { views } = useServerData();
    const availableTiles = tiles.filter(tile => views.find(view => tile.viewName === view.meta.product));
```
We should be able to simplify things with the above.